omnigibson/sensors/vision_sensor.py

```diff
@@ -63,15 +62,20 @@ class VisionSensor(BaseSensor):
         "normal",
         "seg_semantic",
         "seg_instance",
+        "seg_instance_id",
```

Comment:
Leave a comment on the side here:  # returns visual mesh prim path, and also on the above line.